pkgs/by-name/ms/mslicer/package.nix

```
@@ -0,0 +1,57 @@
+{
+  fetchFromGitHub,
+  lib,
+  libglvnd,
+  libxkbcommon,
+  nix-update-script,
+  rustPlatform,
+  vulkan-loader,
+  wayland,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "mslicer";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "connorslade";
+    repo = "mslicer";
+    rev = finalAttrs.version;
+    hash = "sha256-x46k1O7EqXMEwNATG4b7zHIYaMDVveRiq/Z5KPih0Fo=";
+  };
+
+  cargoHash = "sha256-mRbEwxR6bMkybxe7H1dX4Qa1elGiw/lSSz9sSTtp1zw=";
+  useFetchCargoVendor = true;
+
+  buildInputs = [
+    libglvnd
+    libxkbcommon
+    vulkan-loader
+    wayland
+  ];
+
+  # Force linking to libEGL, which is always dlopen()ed, and to
+  # libwayland-client & libxkbcommon, which is dlopen()ed based on the
+  # winit backend.
+  NIX_LDFLAGS = [
+    "--push-state"
+    "--no-as-needed"
+    "-lEGL"
+    "-lvulkan"
+    "-lwayland-client"
+    "-lxkbcommon"
+    "--pop-state"
+  ];
+
+  strictDeps = true;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "An experimental open source slicer for masked stereolithography (resin) printers.";
```

```suggestion
    description = "Experimental open source slicer for masked stereolithography (resin) printers.";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

> Not start with the definite or an indefinite article.